.eslintignore

```diff
@@ -0,0 +1 @@
+pages/_app.js
```

Comment:
I believe this file was generated by next.js, but eslint doesn't like something in there 🤷🏻‍♀️